cordova-lib/src/plugman/util/metadata.js

```diff
@@ -59,3 +59,10 @@ exports.save_fetch_metadata = function(pluginsDir, pluginId, data) {
     fs.writeFileSync(fetchJsonPath, JSON.stringify(metadataJson, null, 4), 'utf-8');
 };
 
+exports.remove_plugin = function(pluginsDir, pluginId){
```

Comment:
IMHO it's better to rename this similar to other methods in this file (`remove_fetch_metadata` for example)